modules/apps/info/info-api/src/main/java/com/liferay/info/exception/InfoFormValidationException.java

```diff
@@ -57,6 +57,33 @@ public String getLocalizedMessage(Locale locale) {
 
 	}
 
+	public static class ExceedsMaxEntries extends InfoFormValidationException {
+
+		public ExceedsMaxEntries(
+			String messageKey, String objectDefinitionLabel) {
+
+			super(StringPool.BLANK);
+
+			_messageKey = messageKey;
+			_objectDefinitionLabel = objectDefinitionLabel;
+		}
+
+		@Override
+		public String getLocalizedMessage(Locale locale) {
+			return getLocalizedMessage(_objectDefinitionLabel, locale);
+		}
+
+		@Override
+		public String getLocalizedMessage(String fieldLabel, Locale locale) {
+			return LanguageUtil.format(
```

Comment:
I think you should follow `InvalidCaptcha` pattern, so this logic should be moved to `getLocalizedMessage(Locale locale)` and you call this method here.